### DISCUSSION TITLE
How to insert some characters between items after activating them?

### DISCUSSION THREAD

**Discussion by pqnhan** [hluk/CopyQ#2065] [General]
I often use Copyq to select pieces of text that I likes and then combine them into an article.
However sometimes I want to insert bullets into some of the items or add a page divider between them.
Is there a convenient way to do that?

---
Sorry if the question is silly as I am new to Copyq.

**Comment by laur89** [comment]
If I understand it correctly, then what you're asking sounds outside the scope of copyq - in the end of the day it's not a text editor.
Edit: although a configuration item like `multi_item_paste_separator` could plausibly be simple enough.
Sounds like a rather extreme corner case though.

Wouldn't be surprised if it's doable via scripting though.

**Comment by dbareis** [comment]
@pqnhan 

It is quite simple assuming you have some obvious way of detecting where you want to split the text up, I have created a simplistic example to get you started (googles should include "javascript"):

```
copyq:

//--- Very simplistic example ---
var ClipboardText = str( clipboard() )                 //Sample text: 1,2,3,4,5,6,7,8,9,10 
var SplitUp             = ClipboardText.split(',')       //\t = TAB
var NewText          = SplitUp.join('\n')

//--- NewText ----
copy(NewText)
paste()
```

I have also attached it as a command file (zipped) you can import set up to use the hotkey: ctrl+alt+f1


[Discussion #2065 - In copyq, F6, Load Commands.zip](https://github.com/hluk/CopyQ/files/9465395/Discussion.2065.-.In.copyq.F6.Load.Commands.zip)
